guides/pypi.md

```diff
@@ -42,3 +43,36 @@ When publishing a package developed at CFPB, we ask that you do the following:
 - Add the [cfpb](https://pypi.org/user/cfpb/) PyPI user as an owner of the project. This ensures that someone from the organization will always be able to update the package.
 
 One gotcha: once you've uploaded a particular archive filename to PyPI, [you can never ever upload that same filename again](https://github.com/pypa/packaging-problems/issues/74). This means that if you upload, say, a version 1.0.2, and after publishing realize there was some error, you can't delete and re-upload a new corrected archive as version 1.0.2. You have to instead publish the corrected archive as version 1.0.3.
+
+## Publishing Python Packages Automatically with Travis
+
+It is possible to [publish your packages automatically with Travis](https://docs.travis-ci.com/user/deployment/pypi/) when you create a GitHub release. To do this you must add the following to your `.travis.yml` file:
+
+```yml
+deploy:
+  provider: pypi
+  user: cfpb
+  password:
+    secure: <encrypted password for cfpb>
+  distributions: "sdist bdist_wheel --universal"
+  on:
+    tags: true
+```
+
+This tells Travis to publish both an `sdist` and a universal `bdist_wheel` distribution to PyPI as the user `cfpb` with the given password (encrypted using `travis encrypt`) whenever a new git tag is created, which GitHub does when creating a new release. 
+
+If you have multiple targets you can add conditions to the `on` value, for example:
+
+```yml
+on:
+  tags: true
+  condition: $TOXENV=py36-dj111-wag113
+```
+
+This will avoid matrix failures where an distribution cannot be uploaded to PyPI for a release more than once.
+
+Additionally, you can have Travis upload to https://test.pypi.org by specifying that as the server:
```

Comment:
This might be too obvious, but it would be friendly to add an example of when you would use test.pypi. Maybe something like "If you're not ready to publish to the real index, you can have Travis ..."